### ISSUE TITLE
Add battibass as muon-dpg L2

### ISSUE THREAD

**Issue by smuzaffar** [cms-sw/cms-bot#1623]
@battibass, I see that you have requested to be member of @cms-sw/muon-dpg-l2 . The correct way to do this is update https://github.com/cms-sw/cms-bot/blob/master/categories.py#L87 and add `muon-dpg` for your user.

**Comment by battibass** [comment]
@smuzaffar, thanks.
Let me mention one thing before proceeding though: I made the request because I saw muon-dpg being called into question [here](https://github.com/cms-sw/cmssw/pull/35105#issuecomment-913655306).
Now, in the present setup, this implies that the only one that gets warned in cases like this is the DPG office coordinator of the 4 muon DPGs (@mileva).
For specific requests where you want to reach experts for muon packages belonging to multiple DPGs, IMHO, it would instead be more efficient to propagate the messages targeting muon-dpg to people listed into csc/dt/rpc/gem-dpg as well.
Anyhow this is just my personal opinion, may be there is an agreement about how the list is structured that I do not understand.
I would let others, mostly @mileva , comment before taking any action.
